packages/stg-evm-v2/devtools/src/credit-messaging/config.ts

```diff
@@ -17,6 +17,10 @@ export const configurePlanner: CreditMessagingConfigurator = createConfigureNode
     const planner = await sdk.getPlanner()
     if (areBytes32Equal(planner, config.planner)) return []
 
+    if (!config.planner) {
+        return [] // No planner to set, do not overwrite the existing one
+    }
```

Comment:
This line should be placed on the very top so that no logic is executed if the value is nullish.

For the comparison, use `config.planner == null`

Reply:
Sounds good, will do.